api/docs/static/override_sphinx.css

```
@@ -1,26 +1,80 @@
 @import url('https://fonts.googleapis.com/css?family=Public+Sans:300,400,400i,600,700');
 @import url('https://fonts.googleapis.com/css2?family=Reddit+Mono:wght@200..900&display=swap');
 
-/* OT NAV */
+/* OT TOP NAV */
 
 body {
     padding: 0;
     margin: 0;
-    font-family: "Public Sans", "sans-serif";
 }
 
-.highlight-none, .mi, .literal {
-    color: #048f85;
+/* restate button styling */
+.text-brand-blue:hover {
+  --tw-text-opacity: 1;
+  color: rgb(0 108 250 / var(--tw-text-opacity));
+}
+
+.text-white:hover {
+  --tw-text-opacity: 1;
+  color: rgb(255 255 255 / var(--tw-text-opacity));
+}
+
+/* hack out search button */
+
+div.flex.gap-4 div.text-white.flex.items-center.h-full > button {
+    display: none;
+}
+
+/* use Akko from docs site instead of from main site */
+header a.button-sm {
+    font-family: "AkkoPro-Regular", "Akko Pro", "Open Sans", sans-serif !important;
+}
+
+/* OT FOOTER */
+
+footer {
+    padding: 0;
+}
+
+.footer-app-button:hover {
+    color: #E8E9EA;
+}
+
+/* restate newsletter form styling */
+
+footer input.hs-input {
+  margin: 0;
+  max-width: 100%;
+  width: 100%;
+  height: 42px;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax

```suggestion
padding: 0.5rem 1rem;
```